landoapi/models/secapproval.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Models related to the sec-approval process.
+
+See See https://wiki.mozilla.org/Security/Bug_Approval_Process.
+"""
+
+from sqlalchemy.dialects.postgresql import JSONB
+
+from landoapi.storage import db
+
+
+class SecApprovalRequestEvent(db.Model):
```

Comment:
"RequestEvent" is jumping out as a little redundant to me, `SecApprovalRequest` would be just as descriptive, no?